library/eclipse-temurin

```diff
@@ -257,34 +257,34 @@ GitCommit: f06ea446cee4810ab29df927d2d4c53f6270a5cd
 Directory: 17/jdk/centos
```

Comment:
Were the 17 jdk/jre non-windows images also supposed to be updated?

Just to be clear about the consequences of still having these versions with `SharedTags: 17.0.1_12-jdk` is that the manifest for those tags will be updated to no longer include the Windows images of that version (similar to the issue we just had in https://github.com/docker-library/python/issues/681)..

Reply:
Sorry, yes just spotted that! I'll get the JDK17 Linux dockerfiles added too